.ci/scripts/run-gradle.sh

```diff
@@ -37,4 +37,4 @@ if [[ "$MAX_WORKERS" == "0" ]]; then
 fi
 
 set -e
-$GRADLEW -S --max-workers=$MAX_WORKERS $@
+$GRADLEW -S --max-workers=$MAX_WORKERS --continue $@
```

Comment:
Did you mean to leave this here as well?

Reply:
I think I'll remove it from this PR and we can have a separate discussion as to whether we should do this across CI in general.